include/vfio-user.h

```diff
@@ -67,6 +67,7 @@ enum vfio_user_command {
     VFIO_USER_DMA_WRITE                 = 12,
     VFIO_USER_DEVICE_RESET              = 13,
     VFIO_USER_DIRTY_PAGES               = 14,
+    VFIO_USER_DEVICE_FEATURE            = 15,
```

Comment:
In your commit message, it'll be helpful if you can describe where we differ from vfio, if we do, and why, so we can understand context a little more.